pyproject.toml

```diff
@@ -101,6 +101,7 @@ export = [
     "openvino>=2024.0.0", # OpenVINO export
     "tensorflow<=2.13.1; python_version <= '3.11'", # TF bug https://github.com/ultralytics/ultralytics/issues/5161
     "tensorflowjs>=3.9.0; python_version <= '3.11'", # TF.js export, automatically installs tensorflow
+    "flatbuffers>=23.5.26,<100", # update old 'flatbuffers' included inside tensorflow package
```

Comment:
@lakshanthad I'm not sure if this line is causing the Dockerfile deploy errors following this PR. I may have to remove it for the time being to get 8.2 out in time.

Reply:
This should not be the issue. Because this change was done a long time ago and the other CI were passing at the time @glenn-jocher. Are you sure this is causing the issue?

Reply:
If this is really the issue, then we can add this for only 'aarch64' @glenn-jocher